src/cljc/athens/common_events/resolver/atomic.cljc

```diff
@@ -17,15 +17,14 @@
 
 (defmethod resolve-atomic-op-to-tx :block/new
   [db {:op/keys [args]}]
-  (let [{:keys [block-uid
-                position]}              args
-        {:keys [ref-uid
-                ref-name
-                relation]}              position
+  (let [{:block/keys [uid position]}    args
+        {relation  :relation
+         ref-uid   :block/uid
+         ref-title :page/title}         position
```

Comment:
What is the meaning of `ref`  in these cases? 

`ref` to me at first glance seems like we are talking about the `block ref`'s ref, are we not using `ref` in multiple scenerios?

Reply:
It's the page/block referred to in position. Events at this level almost never talk about block-refs, and the athens protocol itself doesn't know about refs either.